README.md

```
@@ -1,48 +1,144 @@
 
-## ブランチ運用について
+冷蔵庫にある食材から、AI が最適な料理レシピを提案する Web アプリケーションです。
 
-- `main` を安定開発ブランチとして位置づけ、それとは別にデプロイ用環境ブランチとして `production` を別途置く
-- `feature` → `main` で日常的に統合
-- 適宜 `main` → `production` をマージしてリリース
-- `production` は `hotfix` 直マージ可
+## 🌐 公開 URL
 
-※ GitLab Flow を意識した環境ブランチ型の運用の簡易版です。
+https://fridge-chef-tau.vercel.app/
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)